frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -284,13 +266,11 @@ pods:
           size: {{COORDINATOR_NODE_DISK}}
         cmd: |
           set -e
-
```

Comment:
See the comment above regarding the whitespace.

Reply:
done